.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -29,15 +29,14 @@ body:
   attributes:
     label: Which MonoGame platform are you using?
     options:
+      - N/A
```

Comment:
N/A is usually found at the bottom of the list.

Reply:
Moved to the bottom, thanks